ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -130,6 +130,12 @@ def ocs_install_verification(
             post_upgrade_verification
         ) else 1
 
+    # With 4.4 OCS cluster deployed over Azure, RGW is the default backingstore
+    if float(
+        config.ENV_DATA['ocs_version']
+    ) in [4.4, 4.5] and config.ENV_DATA.get('platform') == constants.AZURE_PLATFORM:
```

Comment:
There should be no RGW on Azure with OCS 4.5.

Reply:
but after the upgrade to 4.5, there will

Reply:
then we need to update this check to be valid only if cluster is after upgrade right?

Reply:
@mbukatov @petr-balogh right, will fix

Reply:
this is already being done

> > @mbukatov not sure this is actually a workaround, as RGW will continue to be the default backingstore in 4.4
> 
> It is change in ocs-ci code to overcome known product BZ 1825994 _RGW pods should not be started in cloud deployments like Azure, GCP_ - so it seems like [a workaround](https://ocs-ci.readthedocs.io/en/latest/docs/workarounds.html) to me. But we don't plan to remove it as the checks are version specific, so we don't have to create issue to remove it.

Exactly